package.json

```diff
@@ -54,10 +54,12 @@
     "site": "bundle exec jekyll serve",
 
     "lint": "tslint --project tsconfig.json -c tslint.json --type-check",
-    "test": "npm run tsc && npm run test:only && npm run lint",
+    "test": "npm run build:only && npm run test:only && npm run test:runtime && npm run lint",
     "posttest": "npm run schema && npm run data && npm run mocha:examples",
-    "test:nocompile": "npm run test:only && npm run lint && npm run mocha:examples",
+    "test:nocompile": "npm run test:only && npm run test:runtime && npm run lint && npm run mocha:examples",
     "test:only": "nyc --reporter=html --reporter=text-summary npm run mocha:test",
+    "test:runtime": "TZ=America/Los_Angeles wdio wdio.conf.js",
+    "test:runtime:generate": "rm -Rf test-runtime/resources && VL_GENERATE_TESTS=true npm run test:runtime",
```

Comment:
Maybe it's worth updating PULL_REQUEST_TEMPLATE.md and CONTRIBUTING.md accordingly?

I also wonder if we should name this more similar to `build:examples` so the similarity is clearer.  (I don't have a good name to suggest though.)